tests/core/test_import.py

```diff
@@ -1,4 +0,0 @@
-
-
-def test_import():
-    import <MODULE_NAME>  # noqa: F401
```

Comment:
This is actually a nice smoke test to keep around to detect really high level errors (that would admittedly manifest in other tests too)